avocado_vt/plugins/vt_bootstrap.py

```diff
@@ -96,14 +96,14 @@ def configure(self, parser):
                             help=("The architecture of the distro to be used when "
                                   "generating the host configuration entry."))
 
-    def run(self, args):
+    def run(self, config):
```

Comment:
we need to use `args` itself else rebase the PR with commit 39f4ba79 and change the below lines to `config`,
https://github.com/avocado-framework/avocado-vt/blob/1efb1c3722918ef517670c4a518c03dfde250d9f/avocado_vt/plugins/vt_bootstrap.py#L107-L108

It breaks the framework with,
```
Avocado crashed:
Traceback (most recent call last):
  File "/usr/local/bin/avocado", line 4, in <module>
    __import__('pkg_resources').run_script('avocado-framework==70.0', 'avocado')
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 666, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1462, in run_script
    exec(code, namespace, namespace)
  File "/usr/local/lib/python3.7/site-packages/avocado_framework-70.0-py3.7.egg/EGG-INFO/scripts/avocado", line 78, in <module>
    sys.exit(app.run())
  File "/usr/local/lib/python3.7/site-packages/avocado_framework-70.0-py3.7.egg/avocado/core/app.py", line 91, in run
    return method(self.parser.config)
  File "/usr/local/lib/python3.7/site-packages/avocado_framework_plugins_vt-70.0-py3.7.egg/avocado_vt/plugins/vt_bootstrap.py", line 107, in run
    if isinstance(self.args, dict):
AttributeError: 'VTBootstrap' object has no attribute 'args'
```